### PR TITLE
(branch deleted) Fixes issue #251 : App Crashing on touching near right and bottom border in camera fragment

### PR DESCRIPTION
**Pull request by vinaysajjanapu**
Fixes issue #251 
Changes: Previously, for calculating focus area fixed values(2000x1000) were used. So I modified the function such that values = device's screen size and it solved the issue.

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @vinaysajjanapu we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/vinaysajjanapu/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Abhi2424shek** [comment]
Can you properly rebase pls?

**Comment by vinaysajjanapu** [comment]
I hope it is done now

File: gradle.properties [hunk 1]
```diff
@@ -23,4 +23,4 @@ org.gradle.jvmargs=-XX:MaxPermSize=1024m -XX:+CMSClassUnloadingEnabled -XX:+Heap
 org.gradle.parallel=true
 # When set to true the Gradle daemon is used to run the build. For local developer builds this is our favorite property.
 # The developer environment is optimized for speed and feedback so we nearly always run Gradle jobs with the daemon.
-org.gradle.daemon=true
+org.gradle.daemon=true
```
Comment (Abhi2424shek) [inline gradle.properties]: Please don't remove the newline at EOF.

Comment (vinaysajjanapu) [inline gradle.properties]: What's your reason for this

**Comment by vinaysajjanapu** [comment]
What's the problem? Can't it be merged?

**Comment by vinaysajjanapu** [comment]
Deleted my fork to add again because it messed up while rebasing.

**Pull request closed without merging**